### DISCUSSION TITLE
리더보드 조회 이슈 (프로젝트 세팅 창)

### DISCUSSION THREAD

**Discussion by lhj5709** [naverz/zepeto-studio-kor#502] [Issue Report (Legacy)]
이번엔  아래와 같은 로딩창만 뜨면서 리더보드가 조회되지 않는 현상이 있습니다...

이런 문제 겪으신 분 이나 해결방법 있을까요 ?

![image](https://user-images.githubusercontent.com/20422451/158740398-270efc59-e0b9-4eca-bc48-fc2b25604d38.png)

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요. #424 와 동일한 현상이며
수정 버전이 올라오기 전까지는 아래 댓글에 안내드린 방법대로 진행하시면 사용 가능합니다 
https://github.com/naverz/zepeto-studio-kor/discussions/424#discussioncomment-2293491

**Reply by lhj5709** [reply]
![image](https://user-images.githubusercontent.com/20422451/158749653-ef2112f3-821d-4db0-879c-73cb2fd30cef.png)

링크에 있는 방법대로 하니 무한 로딩은 없어졌는데요  들어가서 리더보드를 추가할 경우 위와같은 에러가 나오면서 생성이 안됩니다..
그리고 기존에 사용하던 리더보드들도 없어졌습니다
